persistent-mysql/Database/Persist/MySQL.hs

```diff
@@ -795,14 +808,26 @@ getColumn connectInfo getter tname [ PersistText cname
 
         cntrs <- liftIO $ with (stmtQuery stmt vars) (\src -> runConduit $ src .| CL.consume)
         pure $ case cntrs of
-            [] ->
-                Nothing
-            [[PersistText tab, PersistText ref, PersistInt64 pos, PersistText onDel, PersistText onUpd]] ->
+            [] -> Nothing
```

Comment:
This would arguably be easier to review if it weren't so inlined. A comment at the top or breaking it out into a named function would make it much clearer what the intended semantics are